README.md

```diff
@@ -1,24 +1,261 @@
+    ```
+
+Run Linters
+  - You can also check linter errors
+  - Make sure Rubocop and stylelint are installed
+     Install Rubocop, run the following command:
+    ```
+    gem install --no-document rubocop -v '>= 1.0, < 2.0'
+    ```
+    Install Stylelint, run the following command:
+    ```
+    npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+    ```
+  - Now, run the following:
+    ```
+    npx stylelint "**/*.{css,scss}"
+    ```
+### Usage - Start Locally <a name="usage"></a>
+
+To run the project locally, execute the following command:
+
+- Start and login to the database server (for Windows): 
+  ```
+  run psql
+  ```
+  Or start PgAdmin
+
+- Launch the app with the following command to start the rails server:
+  ``
+  rails s
+ ``
+ 
+- Voila! Now, go to the browser and enter: localhost:3000 or 127.0.0.1:3000
+
+### Deployment
+
+You can deploy this project using:
+
+- GitHub Pages from a specific branch (e.g. main).
+- You can also used any cloud service like gcloud or Azure.
+- A deployed version can be found here:
+ [https://proper-rivalee-microverse-student.koyeb.app/]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Authors
+
+👤 **Omet Omeni (Ometman)**
+
+- GitHub: [@ometman](https://github.com/ometman)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ometman/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Improving UI design**
+- **Adding a help information**
+- **Adding contact information**
+- **Full desktop version**
+- **General aesthetic improvement**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ometman/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+As a tech enthusiast, your support is always appreciated. If you like this project please do let us know with your support in any way you see fit.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank especially [Microverse](https://www.microverse.org/) for inspiring this project & project guidelines.
+Thanks to Gregoire Vella for the design (https://www.behance.net/gregoirevella)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I fork this project?**
+
+  - Yes, feel free to fork and knock yourself out :) .
+
+- **Can I share this project with others?**
+  - Yes, you can share this project for any educational purposes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The MIT license link is broken, Please provide a valid link.

![image](https://github.com/ometman/hello-rails-react/assets/68796232/0593d6b1-cacc-424a-bba2-37828839ed61)